inc/link.class.php

```diff
@@ -464,6 +463,21 @@ public function showOrderLinkItem($numref, $data_ref, $canedit, $plugin_order_or
       echo "<br>";
    }
 
+   public function getSerialItem($data){
+      global $DB;
+      $itemtype = strtolower($data['itemtype']);
+      $itemtype_s = $itemtype."s";
+      if ($itemtype_s!="pluginorderothers") {
+         $id = $data['items_id'];
+         $query = "SELECT serial FROM glpi_".$itemtype_s." WHERE id= ".$id;
```

Comment:
It would be better to use querying syntax as described [here](https://glpi-developer-documentation.readthedocs.io/en/master/devapi/database/dbiterator.html) instead of using raw SQL.